server/src/main/java/io/druid/server/AsyncQueryForwardingServlet.java

```diff
@@ -138,12 +138,9 @@ public void init() throws ServletException
   {
     super.init();
 
-    // separate client with more aggressive connection timeouts
-    // to prevent cancellations requests from blocking queries
+    // Note that httpClinetProvider is setup to return same HttpClient instance on each get() so
+    // it is same http client as that is used by parent ProxyServlet.
     broadcastClient = httpClientProvider.get();
-    broadcastClient.setConnectTimeout(CANCELLATION_TIMEOUT_MILLIS);
```

Comment:
What's setting `setConnectTimeout` now? The old comment said it wanted a "separate client with more aggressive connection timeouts" but I don't see anything setting connection timeouts anymore.